cmake/cudawrappers-helper.cmake

```
@@ -45,7 +51,7 @@ function(target_embed_source target input_file)
       ld ARGS -r -b binary -A ${CMAKE_SYSTEM_PROCESSOR} -o
       "${CMAKE_CURRENT_BINARY_DIR}/${NAME}.o" ${input_file_inlined_relative}
     WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
-    DEPENDS ${input_file} ${input_file_inlined}
+    DEPENDS ${input_file} ${input_file_inlined} ${include_files}
```

I don't think that `${input_file_inlined}` is still needed here:
```suggestion
    DEPENDS ${input_file} ${include_files}
```